hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionHealthProblem.java

```diff
@@ -0,0 +1,116 @@
+package pl.allegro.tech.hermes.api;
+
+public enum SubscriptionHealthProblem {
```

Comment:
enums are not flexbile/extensible at all. if i want to make those parameters accessible via configuration, i need to refactor the whole thing to accept configuraion somehow, i don't care if enum is close to the health problem calculator